dataplane/src/vpc/prefix_trie.rs

```diff
@@ -0,0 +1,92 @@
+// SPDX-License-Identifier: Apache-2.0
```

Comment:
We should probably just drop this code in favor of the [`iptrie` crate](https://crates.io/crates/iptrie), which I think @Fredi-raspall is using for the FIB.  Eventually, we'd replace `iptrie` with a tree bitmap that we implement and optimize when `iptrie` becomes a meaningful bottleneck.

Reply:
The PR currently has no functional change. Do you want me to remove the prefix_trie code we have and rework the PIF table to use `iptrie` in this PR? I can take a look into it tomorrow, if this is what you suggest.

Reply:
I think it is worth the effort and shouldn't be too difficult to move, I think.  If it takes more than 1/2 a day, then we can discuss further.

Reply:
Switching to iptrie as a preparatory PR in https://github.com/githedgehog/dataplane/pull/230